src/components/Incidents/IncidentsHeader/IncidentsHeader.jsx

```diff
@@ -2,10 +2,21 @@ import * as React from 'react';
 import IncidentsChart from '../IncidentsChart/IncidentsChart';
 import AlertsChart from '../AlertsChart/AlertsChart';
 
-export const IncidentsHeader = ({ alertsData, incidentsData, chartDays }) => {
+export const IncidentsHeader = ({
+  alertsData,
+  incidentsData,
+  chartDays,
+  chooseIncident,
```

Comment:
I think this might be better named as `selectedIncident`

Reply:
ok then is an action, in that case it might be better `onSelectIncident`

Reply:
I adjusted it according to the other comment